test/factory.js

```diff
@@ -122,24 +133,30 @@ describe("Test Wallet Factory", function () {
         });
 
         it("should fail to create when there is no modules", async () => {
+            let label = "wallet" + index; 
             let modules = [];
-            await assert.revert(factory.from(deployer).createWallet(owner.address, modules, NO_ENS, { gasLimit: 200000 }), "should fail when modules is empty");
+            await assert.revert(factory.from(deployer).createWallet(owner.address, modules, label), "should fail when modules is empty");
+        });
+
+        it("should fail to create when there is no ENS", async () => {
+            let modules = [module1.contractAddress, module2.contractAddress];
+            await assert.revert(factory.from(infrastructure).createWallet(owner.address, modules, NO_ENS), "should fail when ENS is already used");
```

Comment:
All `revert` failures should be checked for the respective error message back, rather than a general failure of the transaction. It's the only way to check that the failure we're testing for is the one that actually occurred.

Reply:
Done. I replaced `assert.revert` by `assert.revertWith` which is now available with etherlime.

We should propagate this change to all tests that use `assert.revert`.